openapi/components/schemas/users/update-user-payload.json

```diff
@@ -0,0 +1,115 @@
+{
+  "UpdateUserRequestModel": {
+    "type": "object",
+    "properties": {
+      "phoneNumbers": {
+        "type": "array",
+        "items": {
+          "type": "string",
+          "pattern": "^[+]?[0-9]{1,3}?[0-9]{10}$"
+        }
+      },
+      "name": {
+        "type": "string"
+      },
+      "bloodGroup": {
+        "type": "string",
+        "enum": [
+          "A+",
+          "A-",
+          "B+",
+          "B-",
+          "AB+",
+          "AB-",
+          "O+",
+          "O-"
+        ]
+      },
+      "lastDonationDate": {
+        "type": "string",
+        "format": "date-time"
+      },
+      "height": {
+        "type": "number"
+      },
+      "weight": {
+        "type": "number"
+      },
+      "gender": {
+        "type": "string",
+        "enum": [
+          "Male",
+          "Female",
+          "Other"
+        ]
+      },
+      "dateOfBirth": {
+        "type": "string",
+        "format": "date"
+      },
+      "age": {
+        "type": "number"
+      },
+      "area": {
+        "type": "string"
+      },
+      "city": {
+        "type": "string"
+      },
+      "latitude": {
```

Comment:
Why has the user a single lat and long? A user can use multiple places for donation acceptance.

Reply:
Updated in 7510b59ea5622893a67d478c91caa3b141d55eea